<!-- -*- mode: XML -*- -->
<launch>    


     <arg name="scenario" default="scenario_1"/> 
    
     <arg name="SFDP_file_url" value="$(find SRVSS)/SFDP/test.SFDP"/>
     <arg name="SFV_file_url" value="$(find SRVSS)/SFV/$(arg scenario).SFV"/>
     <arg name="resource_file_url" value="$(find SRVSS)/resource/resource_robil2.xml"/>
     <arg name="scenarios_folder_url" value="$(find SRVSS)/scenarios/$(arg scenario)"/>


     <node pkg="SRVSS" type="mainGen" name="SRVSS_gen_node" args="-gen $(arg SFDP_file_url) $(arg SFV_file_url) $(arg resource_file_url)" />
     <node pkg="SRVSS" type="mainGen" name="SRVSS_run_node" args="-run $(arg SFV_file_url) $(arg scenarios_folder_url) $(arg resource_file_url)" />    


</launch>
